nemo/core/classes/modelPT.py

```diff
@@ -166,7 +166,7 @@ def save_to(self, save_path: str):
             self.__make_nemo_file_from_folder(filename=save_path, source_dir=tmpdir)
 
     @classmethod
-    def restore_from(cls, restore_path: str):
+    def restore_from(cls, restore_path: str, *args: Any, **kwargs: Any):
```

Comment:
why? could you please change this back to the original signature

Reply:
To facilitate for optional root_dir , else this is not necessary

Reply:
@nithinraok Dont pass root dir like this. Manipulate your own config and your code if needed but dont edit signature of core components which is shared across multiple models. Also, *args and **kwargs will not be allowed in any case as it breaks graphs in some form.